<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsCloudRecognitionForm</class>
 <widget class="QWidget" name="QgsCloudRecognitionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" rowminimumheight="0,0,0,0,0,0,0,0" columnminimumwidth="0,0,0,0,0">
     <item row="3" column="2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>Add &gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addAllButton">
         <property name="text">
          <string>Add all &gt;&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="text">
          <string>Delete &lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="text">
          <string>Clear &lt;&lt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="browseButton">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="5">
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="searchLineEdit"/>
     </item>
     <item row="5" column="1" colspan="3">
      <widget class="QLineEdit" name="destinationDirLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="destinationDirLabel">
       <property name="text">
        <string>Destination folder</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="addFilesButton">
       <property name="text">
        <string>Add to list...</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3" colspan="2">
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QPushButton" name="runButton">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="searchResultsLabel">
       <property name="text">
        <string>Available data</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="selectedDataLabel">
       <property name="text">
        <string>Selected data</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3" colspan="2">
      <widget class="QListWidget" name="selectedDataListWidget"/>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QListWidget" name="searchResultsListWidget"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
